xml/article_sap_trento.xml

```diff
@@ -962,45 +985,49 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
         <screen>&trentoagent.prompt;sudo systemctl stop trento-agent</screen>
       </step>
       <step>
-        <para>Confirm the &t.agent; is down:</para>
-        <screen>&trentoagent.prompt;sudo systemctl status trento-agent</screen>
+        <para>
+          Install the package:
+        </para>
+        <screen>&trentoagent.prompt;sudo zypper ref
+&trentoagent.prompt;sudo zypper install trento-premium</screen>
       </step>
+      <!--<step>
+        <para>
+          Open the configuration file <filename>/etc/trento/agent.yml</filename>
+          and uncomment (<literal>#</literal>) the entries for
+          <parameter>ssh-address</parameter> and <parameter>collector-host</parameter>
+          and update the values appropriately.
+        </para>-->
+      <!--</step>-->
       <step>
-        <para>Update &t.agent;, using the &t.agent; host local IP and
-          the &t.server; host local IP:</para>
-        <screen>&trentoagent.prompt;curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install-agent.sh | sudo bash -s - --agent-bind-ip=<replaceable>TRENTO_AGENT_IP</replaceable> --server-ip=<replaceable>TRENTO_SERVER_IP</replaceable></screen>
+        <para>
+          Start the &t.agent;:
+        </para>
+        <screen>&trentoagent.prompt;sudo systemctl enable --now trento-agent</screen>
       </step>
```

Comment:
AFAIK this should be:

systemctl restart trento-agent

Reply:
Ok, done